controller/ingress_entry.go

```diff
@@ -50,6 +50,9 @@ func (e IngressEntry) validate() error {
 	if e.ServiceAddress == "" {
```

Comment:
I think it might be cleaner to have the check here as we treat an empty string and `None` the same

Reply:
Also, will we see the same issue if `Host` gets set to `None`?

Reply:
Only cluster IP gets set to 'None', I think from error point of view its good to have information about the None, address I propose may be changing this to have message as `Service address is set to None ignoring entry` or on similar lines, may be we can include actual entry in logging on other places as well.

Let me know your thoughts.

Reply:
Yeah I think updating the log message with something like that and leaving it as two checks sounds a good idea :+1:

Reply:
cool that's changed now.